manual/tmva/index.md

```diff
@@ -26,29 +31,77 @@ The package includes:
 
 {% include tutorials name="TMVA" url="tmva" %}
 
-> **Topical manuals**
->
-> For TMVA, topical manuals are available at [Topical Manuals - TMVA]({{ '/topical/#tmva' | relative_url }}).<br>
-> They contain in-depth information about TMVA.
+### Ingesting a dataset for training in TMVA
 
-> **TMVA in the ROOT forum**
->
-> Discuss TMVA in the [ROOT forum](https://root-forum.cern.ch/c/tmva/22){:target="_blank"}.
+A ROOT dataset can be easily ingested for training using a {% include ref
+class="DataLoader" namespace="TMVA" %}. A common usecase in HEP is to have the
+information of physics events split in signal and background, as in the example
+below:
 
-## Using TMVA
-
-The ROOT tutorials for TMVA, available in `$ROOTSYS/tutorials/tmva`, provide example jobs for the training phase and the application of the training results in a classification or regression analysis using the TMVA Reader.
+_**Example**_
 
+{% highlight C++ %}
+void load_data(){
+   // open file and retrieve trees
+   auto inputfile = TFile::Open("http://root.cern/files/tmva_class_example.root");
+   auto signaltree = inputfile->Get<TTree>("TreeS");
+   auto backgroundtree = inputfile->Get<TTree>("TreeB");
+
+   // load trees into TMVA
+   TMVA::DataLoader loader{"mydataloader"};
+   loader.AddSignalTree(signaltree);
+   loader.AddBackgroundTree(backgroundtree);
+}
+{% endhighlight %}
+
+The loaded data can then be passed to the {% include ref class="Factory"
+namespace="TMVA" %} class for training, as shown in
+[this tutorial](https://root.cern/doc/master/tmva003__RReader_8C.html){:target="_blank"}.
+
+## Interoperability with machine learning libraries
+
+Machine learning is a widely researched topic. Many libraries implement
+classification and regression techniques, with a broader scope or in more
+specific fields. These libraries also vary in the programming languages offered
+by their APIs, although it is true that often Python defines a common ground.
+Notable examples include [Keras](https://keras.io/){:target="_blank"},
+[PyTorch](https://pytorch.org/){:target="_blank"} and
+[scikit-learn](https://scikit-learn.org){:target="_blank"}.
+
+### Loading an external model into TMVA
+
+Suppose you have trained a model using one of the external libraries cited above.
+Or maybe you need to use a specific pre-trained model for your usecase. TMVA
+allows loading model files directly in a factory object and use it for example
+to classify a test dataset. In particular, the entire application could be
+written as a single Python script thanks to
+[PyROOT]({{ '/manual/python' | relative_url }}). A good example of this can be
+found in [this tutorial](https://root.cern/doc/master/ClassificationKeras_8py.html){:target="_blank"}.
+
```

Comment:
The example for using a pre-trained model on a test data set should be: 
- https://root.cern.ch/doc/master/ApplicationClassificationKeras_8py.html  (when using old TMVA::Reader interface)
- https://root.cern.ch/doc/master/tmva003__RReader_8C.htmlhttps://root.cern.ch/doc/master/tmva003__RReader_8C.html
  using the new Reader interface.
  
  Probably it would be nice to link only the RReader tutorial, but we need to translate it in Python (and implement a direct conversion numpy->RTensor)

Reply:
Now that I read the other tutorials I get better the source of confusion. in the https://root.cern/doc/master/ClassificationKeras_8py.html the example is just creating a model with `Keras`, then doing the training in TMVA. Your examples are the right ones for pre-trained models instead. I will change this section accordingly, showing only the `ClassificationKeras` example. For the other examples, we can do them in another PR

Reply:
Agree, we can add example for inference, including also SOFIE, in another PR